### ISSUE TITLE
Some GOG Games are not installable

### ISSUE THREAD

**Issue by Bloodscars** [Heroic-Games-Launcher/HeroicGamesLauncher#3882]
### Describe the bug

Some GOG Games are not installable and I cannot download or import those games if they are locally installed. This does not happen on every game as some games are installable and also able to be imported.

### Add logs

```text
(15:49:26) INFO:    [Legendary]:        Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(15:49:26) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(15:49:26) INFO:    [Connection]:       Connectivity: check-online
(15:49:26) INFO:    [Connection]:       Pinging external endpoints
(15:49:26) INFO:    [Gog]:              Checking for existing gog manifests
(15:49:26) INFO:    [Backend]:          DRM module staus {
  "oimompecagnajdejgnnjijobebaeigek": {
    "status": "new",
    "title": "Widevine Content Decryption Module",
    "version": "4.10.2710.0"
  }
}
(15:49:26) WARNING: [Backend]:          Failed to register protocol with OS.
(15:49:26) INFO:    [Connection]:       Connectivity: online
(15:49:26) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:26) INFO:    [Backend]:          User Not Found, removing it from Store
(15:49:26) INFO:    [Gog]:              Checking if login is valid
(15:49:26) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:26) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(15:49:26) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(15:49:26) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/nile_config /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/nile --version
(15:49:26) INFO:    [Frontend]:         Refreshing undefined Library
(15:49:26) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(15:49:26) INFO:    [Legendary]:        Refreshing library...
(15:49:26) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:26) INFO:    [Backend]:          Frontend Ready
(15:49:26) INFO:    [Gog]:              Getting data of 1207187357: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json import "/run/user/1000/doc/c2c9900f/Pathfinder Wrath of the Righteous"
(15:49:26) INFO:    [Gog]:              Getting GOG library
(15:49:26) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:26) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(15:49:27) INFO:    [Backend]:          
System Information:
CPU: 16x AMD Ryzen 7 5800X3D 8-Core Processor
Memory: 50.43 GB (used: 28.57 GB)
GPUs:
  GPU 0:
    Name: Advanced Micro Devices, Inc. [AMD/ATI] Navi 21 [Radeon RX 6800/6800 XT / 6900 XT]
    IDs: D=73bf V=1002 SD=04f2 SV=1043
    Driver: amdgpu
OS: Pop!_OS 22.04 LTS (linux)

The current system is not a Steam Deck
We are running inside a Flatpak container

Software Versions:
  Heroic: 2.14.1 Flame Emperor Sabo
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.1
  Nile: 1.0.2 Jonathan Joestar

(15:49:27) INFO:    [Backend]:          Checking for current version changelog
(15:49:27) INFO:    [Gog]:              Saved username to config file
(15:49:27) ERROR:   [Gog]:              Unable to get current build of 1207187357
(15:49:27) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:28) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(15:49:28) INFO:    [Winetricks]:       Downloading Winetricks
(15:49:29) INFO:    [DXVKInstaller]:    Updating dxvk to: dxvk-2.4
(15:49:31) INFO:    [WineDownloader]:   Updating wine versions info
(15:49:31) INFO:    [WineDownloader]:   Fetching upstream information...
(15:49:31) INFO:    [Backend]:          Starting the Download Queue
(15:49:32) INFO:    [WineDownloader]:   wine versions updated
(15:49:39) ERROR:   [DownloadManager]:  Downloader: Failed to get headers for https://github.com/doitsujin/dxvk/releases/download/v2.4/dxvk-2.4.tar.gz. 
Error: Error: timeout of 10000ms exceeded
(15:49:39) WARNING: [DXVKInstaller]:    Error when downloading dxvk Error: Failed to get headers
    at downloadFile (/app/bin/heroic/resources/app.asar/build/electron/main.159a68a5.js:136:41)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async installOrUpdateTool (/app/bin/heroic/resources/app.asar/build/electron/main.159a68a5.js:99:198)
(15:49:42) ERROR:   [Gog]:              Failed to get remote config information for 1456460669 : Error: timeout of 10000ms exceeded
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at RedirectableRequest.handleRequestTimeout (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:369:16)
    at RedirectableRequest.emit (node:events:514:28)
    at Timeout.<anonymous> (/app/bin/heroic/resources/app.asar/node_modules/follow-redirects/index.js:210:12)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)
(15:49:44) ERROR:   [Gog]:              Failed to get remote config information for 1112159766 : Error: timeout of 10000ms exceeded
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at RedirectableRequest.handleRequestTimeout (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:369:16)
    at RedirectableRequest.emit (node:events:514:28)
    at Timeout.<anonymous> (/app/bin/heroic/resources/app.asar/node_modules/follow-redirects/index.js:210:12)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)
(15:49:45) ERROR:   [Gog]:              Failed to get remote config information for 1207187357 : Error: timeout of 10000ms exceeded
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at RedirectableRequest.handleRequestTimeout (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:369:16)
    at RedirectableRequest.emit (node:events:514:28)
    at Timeout.<anonymous> (/app/bin/heroic/resources/app.asar/node_modules/follow-redirects/index.js:210:12)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)
(15:49:45) INFO:    [Gog]:              Saved games data
(15:49:45) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(15:49:45) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/bloodscars/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
```

### Steps to reproduce

1. Open Heroic Games Launcher
2. Click on a game that has the  ⃠ logo on it
3. Error: This game doesn't appear to be installable. Check downloadable content on https://gog.com/account

### Expected behavior

To be able to download GOG Games that are available on my account.

### Screenshots

![image](https://github.com/user-attachments/assets/53048d99-4f6c-4442-8a74-b968a9fb1590)

### Heroic Version

Latest Stable

### System Information

- Pop OS 22.04
- Heroic Version 2.14.1 Flatpak

**Comment by PervWizz** [comment]
I'm affected by the same problem. It becomes more severe the more games you have on GoG. I have >2k titles and >90% of them are not installable. The ones that I can install change each reset of HGL.

From what I could google this problem was already solved and fix is merged however there hasn't been a release since April so everybody using official releases is still affected.

When can we expect 2.15 or 2.14.2 to be out?

**Comment by DelayedHappiness** [comment]
Same issue here. all my games aren't even appearing on the list

**Comment by hardpenguin** [comment]
This keeps happening and this is so annoying. I don't even use the flatpak version.

**Comment by CodesInChaos** [comment]
Probably a duplicate of #3636. It's fixed on master, but apparently not important enough to actually release.

**Comment by imLinguin** [comment]
@CodesInChaos we plan to release this week, there are some other pending changes we want in this release

**Comment by PervWizz** [comment]
@imLinguin Is the release still coming this week? I looked at the [milestone](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/milestone/12) ad there has not been a lot of progress.

**Comment by PervWizz** [comment]
I downloaded AppImage from [here](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/10305311043?pr=3663) (This is a build from MR connected to 2.15.0 release) and confirmed that this will be solved on 2.15.0.

**Comment by arielj** [comment]
closing since 2.15.0 was just released and it solves this
